### PR TITLE
CIO-271 Add format script with prettier on all js files

### LINKED ISSUE #271: Add a 'format' script to package.json
We should have a format script that could be called with `npm run format` to run prettier on the project source files.

Prettier and eslint are already setup. I am not 100% sure `npm run lint:fix` script will actually fix everything prettier will.

### PR DESCRIPTION
**Pull request by DonaldLika**
Fixes #271

### CODE REVIEW

**Comment by jwu910-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Changes were made to package.json, but not to package-lock.json - <i>Perhaps you need to run `npm install`? 
Package Lock files can be commited with  the message `CIO-#<ISSUE_NUMBER> Autogenerate package-lock`</i>
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by DonaldLika** [comment]
@jwu910-danger I tried to assign someone to merge this PR but there is no way to do that

**Comment by jwu910** [comment]
@donaldlika I’ll take care of assigning. Looking into auto assigning so it won’t be an issue. Thanks.

One thing regarding your commits, can you just sort the scripts alphabetically and send that in? Thanks!

**Comment by jwu910** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: package.json [hunk 1]
```diff
@@ -12,15 +12,16 @@
...
     "lint": "eslint --config .eslintrc.json ./",
     "precise-commits": "precise-commits",
     "precommit": "npm run precise-commits",
     "prepublishOnly": "npm run build",
     "start": "node index.js",
     "test": "npm run lint",
-    "watch": "gulp watch",
-    "format": "prettier --write **/*.js"
+    "watch": "gulp watch"
+
```
Comment (jwu910) [inline package.json]: There's an extra line here we can remove

File: package.json [hunk 2]
```diff
@@ -12,15 +12,16 @@
     "build": "gulp build",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
+    "format": "prettier --write **/*.js",
```
Comment (jwu910) [inline package.json]: This currently is only checking one level in. Can we write a path glob that gets all of our source files?

We'll also want to ignore `dist/`

Comment (DonaldLika) [reply package.json]: @jwu910  The path gets all js sources in the project not only the first level in.
                  The dist and node_modules directories are added to .prettierignore

Comment (jwu910) [reply package.json]: Maybe I missed something. I made some changes to files inside src/utils, and they were not corrected with `npm run format`.

Were you able to get it working?

Comment (DonaldLika) [reply package.json]: Hmm thats weird,i just checked it and prettier is applied on all directories

Comment (jwu910) [reply package.json]: I just found that the glob needs to be wrapped in quotes. What OS are you on?

Comment (DonaldLika) [reply package.json]: Done already

File: package.json (hunk 1 above)
Comment (jwu910) [reply package.json]: @DonaldLika Can you remove this extra line? Once we get this change fixed, I'll move forward and get this merged.

Thanks!

**Comment by jwu910** [comment]
Merged, thanks!

**Pull request merged**